CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## Release 1.3.0
+### Summary
+This release is a roll up of minor bugfixes.
+
+### Bugfixes
+- (MODULES-9004) Resize XFS file system by mount point, not by volume
+
+###Added
```

Comment:
Formatting of this header is broken. There should be a space after `###`. Additionally, the `Added` section should be above the `Bugfixes` section.